### PR TITLE
Terminology page updated to include where used and transferred to table

### PR DESCRIPTION
**Pull request by JayMurdoch**
Change to indicate where each CodeSystem and ValueSet is used.
Contents of table manually updated using information from valuset-ref-all-list and codesystem-ref-all-list output files.

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-patient-intro.md [hunk 1]
```diff
@@ -1,55 +1,7 @@
 ### Usage Notes
 
 **Profile specific implementation guidance:**
-- This profile supports patient gender identity aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) using the [Individual Gender Identity extension](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) and the  [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set.
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-patient-intro.md:4]: Revert this change.

File: input/intro-notes/StructureDefinition-au-practitioner-intro.md [hunk 2]
```diff
@@ -2,57 +2,8 @@
...
 - See [Ahpra Data Guidance](guidance.html#ahpra-data-guidance) for detailed guidance on representing Ahpra-sourced data.
 - A tertiary qualification or professional membership (non-Ahpra-sourced data) is represented by `Practitioner.qualification`
   - If none of the codes from the preferred value set are suitable then at least text should be sent in `Practitioner.qualification.code`
 - See each Identifier profile page for guidance related to that identifier type.
-- This profile supports the practitioner's gender identity aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) using the [Individual Gender Identity extension](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) and [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set.
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-practitioner-intro.md:9]: Revert the change to this file.

File: input/intro-notes/StructureDefinition-au-relatedperson-intro.md [hunk 3]
```diff
@@ -1,53 +1,5 @@
 ### Usage Notes
 
 **Profile specific implementation guidance:**
-- See each Identifier profile page for guidance related to that identifier type.
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-relatedperson-intro.md:4]: Revert the change to this file.

File: input/pagecontent/guidance.md [hunk 4]
```diff
@@ -353,3 +353,54 @@ This guidance matches Ahpra data items to the corresponding element in a Practit
         <td>Practitioner.qualification.extension:ahpraregistration-details.ahpraNotation</td>
     </tr>
 </table>
+
+### Gender Identity and related concepts
+- This implementation guide supports patient gender identity aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) using the [Individual Gender Identity extension](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) and the  [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set.
```
Comment (dt-r) [inline input/pagecontent/guidance.md:358]: Revert the change to this file.

File: input/pagecontent/terminology.md [hunk 5]
```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
...
-- [Separation Mode](https://healthterminologies.gov.au/fhir/ValueSet/separation-mode-1)
-- [Service Type](https://healthterminologies.gov.au/fhir/ValueSet/service-type-1)
-- [Specimen Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-type-1)
-- [Specimen Collection Procedure](https://healthterminologies.gov.au/fhir/ValueSet/specimen-collection-procedure-1)
-- [Specimen Container Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-container-type-1)
-- [Substance](https://healthterminologies.gov.au/fhir/ValueSet/substance-1)
-- [Vaccination Target Disease](https://healthterminologies.gov.au/fhir/ValueSet/vaccination-target-disease-1)
+|ValueSet|Profile where used|Published|
+|---|---|---|
+|[Accession Number Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-accession-number-type.html)|[AU Accession Number](StructureDefinition-au-accessionnumber.html)|AU Base|
```
Comment (dt-r) [inline input/pagecontent/terminology.md:12]: For any AU Base value set use the relative URL not absolute, e.g. 
instead of: https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-accession-number-type.html
use: ValueSet-accession-number-type.html

File: input/pagecontent/terminology.md [hunk 6]
```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
...
+|[Clinical Finding](https://healthterminologies.gov.au/fhir/ValueSet/clinical-finding-1)|[AU Base Allergy Intolerance](StructureDefinition-au-allergyintolerance.html), [AU Base Condition](StructureDefinition-au-condition.html), [AU Base Diagnostic Imaging Report](StructureDefinition-au-imagingreport.html), [AU Base Diagnostic Report](StructureDefinition-au-diagnosticreport.html), [AU Base Dosage](StructureDefinition-au-dosage.html), [AU Base Pathology Report](StructureDefinition-au-pathologyreport.html)|NCTS|
+|[Clinical Specialty](https://healthterminologies.gov.au/fhir/ValueSet/clinical-specialty-1)|[AU Base Healthcare Service](StructureDefinition-au-healthcareservice.html), [AU Base Practitioner Role](StructureDefinition-au-practitionerrole.html)|NCTS|
+|[Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-jurisdiction.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
```
Comment (dt-r) [inline input/pagecontent/terminology.md:29]: These Value Set names have changed to remove Common. Please update.

File: input/pagecontent/terminology.md [hunk 7]
```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
...
+|[Specimen Container Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-container-type-1)|[AU Base Specimen](StructureDefinition-au-specimen.html)|NCTS|
+|[Specimen Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-type-1)|[AU Base Specimen](StructureDefinition-au-specimen.html)|NCTS|
+|[Substance](https://healthterminologies.gov.au/fhir/ValueSet/substance-1)|[AU Base Substance](StructureDefinition-au-substance.html)|NCTS|
+|[Vaccination Target Disease](https://healthterminologies.gov.au/fhir/ValueSet/vaccination-target-disease-1)|[AU Base Immunisation](StructureDefinition-au-immunization.html)|NCTS|
+{:.grid}
 
 
 ### Code Systems
 
 See the [FHIR terminology section]({{site.data.fhir.path}}terminologies-systems.html) for a complete discussion on code systems and a list of codes system names used in FHIR.  Most code systems used in this guide are defined in the base FHIR specification. The following code systems are unique to this guide and not listed in the base FHIR specification.
```
Comment (dt-r) [inline input/pagecontent/terminology.md:93]: This paragraph doesn't work. As we are making this improvement. Let's replace 'The following code systems are unique to this guide and not listed in the base FHIR specification' with 'The following code systems are used in the above value sets.

File: input/pagecontent/terminology.md [hunk 8]
```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
...
 
+|CodeSystem|ValueSet where used|Published|
+|---|---|---|
+|[ActCode AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-v3-ActCode.html)|[ActEncounterCode - AU Extended](ValueSet-au-v3-ActEncounterCode-extended.html)|AU Base|
```
Comment (dt-r) [inline input/pagecontent/terminology.md:97]: In addition to the code systems in this list add in the others that are used in the above value sets e.g. SNOMED, Identifier Type etc.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments

File: input/pagecontent/terminology.md [hunk 9]
```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
...
-- [Specimen Container Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-container-type-1)
-- [Substance](https://healthterminologies.gov.au/fhir/ValueSet/substance-1)
-- [Vaccination Target Disease](https://healthterminologies.gov.au/fhir/ValueSet/vaccination-target-disease-1)
+|ValueSet|Profile where used|Published|
+|---|---|---|
+|[Accession Number Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-accession-number-type.html)|[AU Accession Number](StructureDefinition-au-accessionnumber.html)|AU Base|
+|[ActEncounterCode - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-v3-ActEncounterCode-extended.html)|[AU Base Encounter](StructureDefinition-au-encounter.html)|AU Base|
+|[Adverse Reaction Agent](https://healthterminologies.gov.au/fhir/ValueSet/adverse-reaction-agent-1)|[AU Base Allergy Intolerance](StructureDefinition-au-allergyintolerance.html)|NCTS|
+|[AMT Medicinal Product and Substances](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-amt-mp-codes.html)|[AU Base Substance](StructureDefinition-au-substance.html)|AU Base|
```
Comment (dbojicic) [inline input/pagecontent/terminology.md:15]: also used in AU Base Medication

File: input/pagecontent/terminology.md [hunk 10]
```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
...
+|[Medicine Item Change](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-medicine-item-change.html)|[AU Medicine List](StructureDefinition-au-medlist.html)|AU Base|
+|[Medicine Substitution Reason](https://healthterminologies.gov.au/fhir/ValueSet/medicine-substitution-reason-1)|[AU Base Medication Dispense](StructureDefinition-au-medicationdispense.html), [AU Base Medication Request](StructureDefinition-au-medicationrequest.html)|NCTS|
+|[MIMS](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-mims.html)|[AU Base Medication](StructureDefinition-au-medication.html), [AU Base Medication Administration](StructureDefinition-au-medicationadministration.html), [AU Base Medication Dispense](StructureDefinition-au-medicationdispense.html), [AU Base Medication Request](StructureDefinition-au-medicationrequest.html), [AU Base Medication Statement](StructureDefinition-au-medicationstatement.html)|AU Base|
+|[Observation Category Codes - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-observation-category-extended.html)||AU Base|
```
Comment (dbojicic) [inline input/pagecontent/terminology.md:62]: Please include Name Context value set (https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-name-context.html), used in Name Context extension (https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-name-context.html) that was only added yesterday.

File: input/pagecontent/terminology.md [hunk 11]
```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
...
+|[Location Type (Physical) AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-location-physical-type.html)|[Location Type (Physical) - AU Extended](ValueSet-au-location-physical-type-extended.html)|AU Base|
+|[Location Type AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-location-type.html)|[ServiceDeliveryLocationRoleType - AU Extended](ValueSet-au-location-physical-type-extended.html)|AU Base|
+|[Medication Type](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-medication-type.html)|[Medication Type](ValueSet-medication-type.html)|AU Base|
+|[Medicine Item Change](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-medicine-item-change.html)|[Medicine Item Change](ValueSet-medicine-item-change.html)|AU Base|
+|[Monthly Index of Medical Specialties (MIMS)](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-mims-external.html)|[MIMS](ValueSet-mims.html)|AU Base|
+|[Observation Category Codes AU](https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-au-observation-category.html)|[Observation Category Codes - AU Extended](ValueSet-au-observation-category-extended.html)|AU Base|
```
Comment (dbojicic) [inline input/pagecontent/terminology.md:116]: Please add Name Context code system, https://build.fhir.org/ig/hl7au/au-fhir-base/CodeSystem-name-context.html.

File: input/intro-notes/StructureDefinition-au-patient-intro.md (hunk 1 above)
Comment (JayMurdoch) [reply input/intro-notes/StructureDefinition-au-patient-intro.md]: Removed, thanks.

File: input/intro-notes/StructureDefinition-au-practitioner-intro.md (hunk 2 above)
Comment (JayMurdoch) [reply input/intro-notes/StructureDefinition-au-practitioner-intro.md]: Removed, thanks.

File: input/intro-notes/StructureDefinition-au-relatedperson-intro.md (hunk 3 above)
Comment (JayMurdoch) [reply input/intro-notes/StructureDefinition-au-relatedperson-intro.md]: Removed, thanks.

File: input/pagecontent/guidance.md (hunk 4 above)
Comment (JayMurdoch) [reply input/pagecontent/guidance.md]: Removed, thanks.

File: input/pagecontent/terminology.md (hunk 6 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:29]: Updated to remove common, thanks.

File: input/pagecontent/terminology.md (hunk 7 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:93]: Replaced, thanks.

File: input/pagecontent/terminology.md (hunk 9 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:15]: Amended, thanks.

File: input/pagecontent/terminology.md (hunk 10 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:62]: Added, thanks.

File: input/pagecontent/terminology.md (hunk 11 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:116]: Added, thanks.

File: input/pagecontent/terminology.md (hunk 9 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:15]: Amended, thanks.

File: input/pagecontent/terminology.md (hunk 7 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:93]: Replaced, thanks.

File: input/pagecontent/terminology.md (hunk 6 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:29]: Amended all instances, thanks.

File: input/pagecontent/terminology.md (hunk 5 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:12]: Amended all instances, thanks.

File: input/pagecontent/terminology.md (hunk 8 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:97]: Added all code systems used by value sets listed, thanks.

File: input/pagecontent/terminology.md [hunk 12]
```diff
@@ -26,44 +26,45 @@ See the [FHIR terminology section]({{site.data.fhir.path}}terminologies-valueset
...
-|[Common AU Recorded Sex or Gender (RSG) Source Document Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-type.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
-|[Common AU Recorded Sex or Gender Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-type.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
+|[AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](ValueSet-rsg-source-document-jurisdiction.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
```
Comment (dt-r) [inline input/pagecontent/terminology.md:29]: The location of the gender and sex value sets including pronouns is their respective extensions not directly to an element in the profile..

File: input/pagecontent/terminology.md [hunk 13]
```diff
@@ -7,93 +7,131 @@ As well as these included definitions this specification typically references [N
...
+|ValueSet|Profile where used|Published|
+|---|---|---|
+|[Accession Number Type](ValueSet-accession-number-type.html)|[AU Accession Number](StructureDefinition-au-accessionnumber.html)|AU Base|
+|[ActEncounterCode - AU Extended](ValueSet-au-v3-ActEncounterCode-extended.html)|[AU Base Encounter](StructureDefinition-au-encounter.html)|AU Base|
+|[Adverse Reaction Agent](https://healthterminologies.gov.au/fhir/ValueSet/adverse-reaction-agent-1)|[AU Base Allergy Intolerance](StructureDefinition-au-allergyintolerance.html)|NCTS|
+|[AMT Medicinal Product and Substances](ValueSet-amt-mp-codes.html)|[AU Base Medication](StructureDefinition-au-medication.html)|AU Base|
+|[Assertion Of Absence](https://healthterminologies.gov.au/fhir/ValueSet/assertion-of-absence-1)|[AU Assertion of No Relevant Finding](StructureDefinition-au-norelevantfinding.html)|NCTS|
+|[AU Time Zone](ValueSet-au-timezone.html)|[Australian Time Zone](StructureDefinition-au-timezone.html)|AU Base|
+|[Australian Immunisation Register Vaccine](https://healthterminologies.gov.au/fhir/ValueSet/australian-immunisation-register-vaccine-1)|[Au Base Immunisation](StructureDefinition-au-immunization.html)|NCTS|
```
Comment (dt-r) [inline input/pagecontent/terminology.md:18]: Fix capitalisation of "Au" to "AU"

File: input/pagecontent/terminology.md [hunk 14]
```diff
@@ -7,93 +7,131 @@ As well as these included definitions this specification typically references [N
...
...es.gov.au/fhir/ValueSet/procedure-1), [Reason for Non Participation in Health Program](https://healthterminologies.gov.au/fhir/ValueSet/reason-non-participation-health-program-1), [Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1), [Reason Vaccine Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-administered-1), [Reason Vaccine Not Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-not-administered-3), [Related Person Relationship Type](https://healthterminologies.gov.au/fhir/ValueSet/related-person-relationship-type-1), [Service Type](https://healthterminologies.gov.au/fhir/ValueSet/service-type-1), [Specimen Collection Procedure](https://healthterminologies.gov.au/fhir/ValueSet/specimen-collection-procedure-1), [Specimen Container Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-container-type-1), [Specimen Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-type-1), [Substance](https://healthterminologies.gov.au/fhir/ValueSet/substance-1), [Vaccination Target Disease](https://healthterminologies.gov.au/fhir/ValueSet/vaccination-target-disease-1)|NCTS, FHIR|
```
Comment (dt-r) [inline input/pagecontent/terminology.md:132]: Remove the term "Australian extension" that is not part of the code system name.

File: input/pagecontent/terminology.md [hunk 15]
```diff
@@ -7,93 +7,131 @@ As well as these included definitions this specification typically references [N
...
+|[IdentifierType](http://terminology.hl7.org/CodeSystem/v2-0203)|[Accession Number Type](ValueSet-accession-number-type.html), [Order Identifier Type](ValueSet-order-identifier-type.html), [hl7VS-identifierType - AU Extended](ValueSet-au-v2-0203-extended.html)|FHIR|
+|[IdentifierType AU](CodeSystem-au-v2-0203.html)|[hl7VS-identifierType - AU Extended](ValueSet-au-v2-0203-extended.html), [DVA Entitlement](ValueSet-dva-entitlement.html)|AU Base|
+|[Individual Healthcare Identifier Record Status](https://healthterminologies.gov.au/fhir/CodeSystem/ihi-record-status-1)|[Individual Healthcare Identifier Record Status](https://healthterminologies.gov.au/fhir/ValueSet/ihi-record-status-1)|NCTS|
+|[Individual Healthcare Identifier Status](https://healthterminologies.gov.au/fhir/CodeSystem/ihi-status-1)|[Individual Healthcare Identifier Status](https://healthterminologies.gov.au/fhir/ValueSet/ihi-status-1)|NCTS|
+|[ISO3166Part1](https://terminology.hl7.org/CodeSystem-ISO3166Part1.html)|[AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](ValueSet-rsg-source-document-jurisdiction.html)|FHIR|
```
Comment (dt-r) [inline input/pagecontent/terminology.md:118]: add in ISO311Part2

**Review by dt-r — CHANGES_REQUESTED** [review]
see inline comments

File: input/pagecontent/terminology.md (hunk 12 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:29]: Noted - amended location to respective extension, thanks.

File: input/pagecontent/terminology.md (hunk 13 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:18]: Fixed, thanks.

File: input/pagecontent/terminology.md (hunk 14 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:132]: Amended, thanks.

File: input/pagecontent/terminology.md (hunk 15 above)
Comment (JayMurdoch) [reply input/pagecontent/terminology.md:118]: 5 code systems added, thanks.

**Review by dt-r — APPROVED** [review]

File: input/pagecontent/terminology.md (hunk 10 above)
Comment (dbojicic) [reply input/pagecontent/terminology.md]: Confirmed.

File: input/pagecontent/terminology.md (hunk 9 above)
Comment (dbojicic) [reply input/pagecontent/terminology.md]: Confirmed.

File: input/pagecontent/terminology.md (hunk 11 above)
Comment (dbojicic) [reply input/pagecontent/terminology.md]: Confirmed.

**Review by dbojicic — APPROVED** [review]
